journal/modernized/12-chapter-9.adoc

```diff
@@ -292,11 +320,13 @@ and that his reign will be only a continuation
 of his brother`'s. The Minister of the Interior,
 whom I saw this moment, says that the Emperor has verbally assured them of the same.
 Let God`'s will be done on earth, as it is in heaven.
-Wishing you well, I am
+Wishing you well, I am.
```

Comment:
Yeah, i see why you did this, but after mulling a bit... Let's remove this period. it's awkward, but the closing below is sort of the continuation of this sentence, and I can't at the moment think of a better way to do it.